packages/connect-form/src/components/connect-form.tsx

```diff
@@ -48,10 +48,18 @@ const descriptionStyles = css({
   marginTop: spacing[2],
 });
 
+const scrollbarWidth = spacing[2];
 const formContentContainerStyles = css({
   padding: spacing[4],
-  overflow: 'scroll',
+  paddingRight: spacing[4] - scrollbarWidth,
+  overflowY: 'scroll',
   position: 'relative',
+  scrollPadding: 0,
+  scrollbarGutter: 'unset',
+  scrollbarWidth: 'none',
```

Comment:
These are not applied in the app due to the electron version we use:

<img width="216" alt="image" src="https://user-images.githubusercontent.com/5036933/150785965-73670c26-cee9-4e72-86c9-e85394e13dbe.png">

```suggestion
```